src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Random random = new Random();
+    final int maxNumber = 100;
```

Comment:
Add access modifiers and [rename according to the convensions](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)